crates/rooch-types/src/address.rs

```diff
@@ -528,4 +528,21 @@ mod test {
             assert_eq!(address, deserialized);
         }
     }
+
+    #[test]
+    pub fn test_from_script() {
```

Comment:
Since it's for signet script test, shall we also add test for regnet or testnet?